plugins/modules/packaging/os/copr.py

```diff
@@ -193,18 +195,25 @@ def _download_repo_info(self):
         Returns:
             Information about the repository.
         """
-        distribution, version = self.short_chroot.split("-")
+        short_chroot_parts = self.short_chroot.split("-")
+        if len(short_chroot_parts) == 3:
+            distribution = short_chroot_parts[0]
+            version = "{0}-{1}".format(short_chroot_parts[1], short_chroot_parts[2])
+        else:
+            distribution, version = short_chroot_parts
         chroot = self.short_chroot
         while True:
             repo_info, status_code = self._get(chroot)
             if repo_info:
                 return repo_info
             if distribution == "rhel":
-                chroot = "centos-stream"
+                chroot = "centos-stream-8"
                 distribution = "centos"
             elif distribution == "centos":
-                if version == "stream":
+                if version == "stream-8":
```

Comment:
Similar question here, since the case `version == "stream"` is no longer covered.